content/docs/other-guides/virtual-dev/getting-started-minikube.md

```diff
@@ -93,15 +93,15 @@ chmod +x setup-minikube.sh
 **KUBEFLOW_TAG** is a tag corresponding to the version to checkout such as {{% kf-latest-version %}} or `master`.
 
 The script asks for some config input as shown below:
-![ConfigInputs](../ConfigInputs.png)
+![ConfigInputs](../../started/ConfigInputs.png)
```

Comment:
The images are still broken - see preview:
https://deploy-preview-1254--competent-brattain-de2d6d.netlify.com/docs/other-guides/virtual-dev/getting-started-minikube/

For the path to the image, it's best to use a relative path that starts with `/docs/...`. While you're here, please would you add styling as well, to make the images look better. The pattern looks like this:

```
<img src="/docs/images/kubeflow-deployment.png" 
  alt="Kubeflow deployment UI"
  class="mt-3 mb-3 border border-info rounded">
```

The above example comes from this file: https://raw.githubusercontent.com/kubeflow/website/master/content/docs/gke/deploy/deploy-ui.md

Reply:
Thanks for your suggestions!